quartodoc/autosummary.py

```diff
@@ -562,11 +567,12 @@ def build(self, filter: str = "*"):
 
         _log.info("Writing docs pages")
         self.write_doc_pages(pages, filter)
+        self.renderer.pages_written()
```

Comment:
@has2k1 WDYT of us...

* renaming this to `._pages_written()` for now, warning in docstring it's internal and experimental.
* passing the builder in here (e.g. `.renderer._pages_written(self)`)
* not setting the builder directly on the renderer

AFAICT the builder is [only used in plotnine in .pages_written()](https://github.com/has2k1/plotnine/pull/706/files#diff-40888a3bc465e5ccae6762cac4b5d8aeae792b4a97f9e120bbbda1303daa7798R702)

Reply:
For now that can work. Though we should soon find a solution for the renderer to know what the key information that the builder digests from the config.

As it stands, the ones that I have so far found useful to know are:

1. `dir` - Lets a renderer know where to create "dark" pages
2. `title` - As a renderer should be rendering the title to the index page.
3. `package` - Allows the renderer to differentiate internal interlinks from external ones. Internal type annotations are a sub-category of this use case.

Reply:
Thanks for weighing in on things the renderer likely needs down the road. I think it's helping for thinking about what exactly is the scope of the renderer? I could see how things about the reference docs at large could come up when rendering (though, we probably want to avoid so much site specific stuff surfacing that the renderer takes on builder jobs, like writing pages documentation).